addons/advanced_ballistics/stringtable.xml

```diff
@@ -48,6 +50,7 @@
             <German>Aktiviert die erweiterte Ballistik</German>
             <Czech>Aktivuje pokročilou balistiku</Czech>
             <Portuguese>Ativa balística avançada</Portuguese>
+            <Hungarian>Engedélyezi a fejlett ballisztikát</Hungarian>
```

Comment:
I´d change "Engedélyezi" into "Aktiválya" feels better for me but i might be a bit rusty.